Branch or Avoton Architecture available

Peda86 opened the pull request:
Hello, i would like to use Zabbix on DS2415+ with Avoton architecture. Is there a branch available?
Many thanks in advance!

DONCIEUX-CAMILLE committed:
Add files via upload

DONCIEUX-CAMILLE committed:
Update to Zabbix-3.4.3 with arch armada38x-6.1

DONCIEUX-CAMILLE committed:
Update to Zabbix-3.4.3 with arch qoriq-6.1

DONCIEUX-CAMILLE committed:
Update to Zabbix-3.4.3 with arch Cedarview

DONCIEUX-CAMILLE committed:
Update to Zabbix-3.4.4 with arch apollolake 6.1

Bug fixed in Zabbix controls

DONCIEUX-CAMILLE committed:
Update to Zabbix-3.4.4 with arch armada38x.

Bug fixed in Zabbix Controls

DONCIEUX-CAMILLE committed:
Update to Zabbix-3.4.3 with arch cedarview-6.1

Bugs fixed in Zabbix Admin Control

DONCIEUX-CAMILLE committed:
Update to Zabbix-3.4.4 with arch qoriq-6.1

Bugs fixed in Zabbix Admin Control

DONCIEUX-CAMILLE committed:
Zabbix 3.4.4 with installer fixed

Thanks to user feedback, installer was fixed.

DONCIEUX-CAMILLE committed:
Zabbix 3.4.4 with installer fixed

Installer fixed
Packages pre-requisites added.

DONCIEUX-CAMILLE committed:
Zabbix 3.4.4 with installer fixed

Installer fixed
Packages pre-requisites added.

DONCIEUX-CAMILLE committed:
Zabbix 3.4.4 on old DSM 4.2

DONCIEUX-CAMILLE committed:
DSM4.2 without SPK depends

DONCIEUX-CAMILLE committed:
Zabbix 3.4.4

DONCIEUX-CAMILLE committed:
Add files via upload

DONCIEUX-CAMILLE committed:
Delete zabbix_apollolake-6.1_3.4.13-1

DONCIEUX-CAMILLE committed:
Delete zabbix_88f6281-4.2_3.4.4-5.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_88f6281-4.2_3.4.4-4.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_apollolake-6.1_3.4.4-3.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_qoriq-6.1_3.4.4-3.spk

DONCIEUX-CAMILLE committed:
Update README.md

DONCIEUX-CAMILLE committed:
Update README.md

DONCIEUX-CAMILLE committed:
Delete zabbix_cedarview-6.1_3.4.3-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_2.4.6-3.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_2.4.5-3.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_2.4.4-3.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_armada38x-6.1_3.4.3-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_cedarview-6.1_3.4.4-3.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_2.4.7-3.spk

DONCIEUX-CAMILLE committed:
Zabbix 4.2.1 with Synology system 6.2

DONCIEUX-CAMILLE committed:
Delete zabbix_88f6281-4.2_3.4.4-6.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_apollolake-6.1_3.4.4-4.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_armada38x-5.2_3.4.3-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_armada38x-6.1_3.4.4-3.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_cedarview-6.1_3.4.4-4.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_3.0.2-4.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_3.0.2-5.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_3.0.4-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_3.2.2-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_qoriq-6.1_3.4.3-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_qoriq-6.1_3.4.4-4.spk

DONCIEUX-CAMILLE committed:
Zabbix 4.2.1 with Synology system 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.2.1 with Synology system 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.2.1 with Synology system 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.2.4 with Evansport 5.2

DONCIEUX-CAMILLE committed:
Delete zabbix_apollolake-6.2_4.2.1-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_armada38x-6.2_4.2.1-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_cedarview-6.2_4.2.1-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_qoriq-6.2_4.2.1-1.spk

DONCIEUX-CAMILLE committed:
Zabbix 4.2.4

DONCIEUX-CAMILLE committed:
Zabbix 4.2.4

DONCIEUX-CAMILLE committed:
Zabbix 4.2.4

DONCIEUX-CAMILLE committed:
Zabbix 4.2.4

DONCIEUX-CAMILLE committed:
Zabbix 4.2.6 for arch Qoriq 6.2

DONCIEUX-CAMILLE committed:
Delete zabbix_qoriq-6.2_4.2.4-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_4.2.4-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_cedarview-6.2_4.2.4-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_apollolake-6.2_4.2.4-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_armada38x-6.2_4.2.4-1.spk

DONCIEUX-CAMILLE committed:
Zabbix 4.2.6 for arch Armada38x 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.2.6 for Arch Evansport 5.2

DONCIEUX-CAMILLE committed:
Zabbix 4.2.6 for Arch Cedarview 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.2.6 for Arch Apollolake 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.2.6 for Arch RTD1296 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.4.0 for Arch Armada38x 6.2

https://www.zabbix.com/whats_new_4_4

DONCIEUX-CAMILLE committed:
Delete zabbix_armada38x-6.2_4.2.6-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_apollolake-6.2_4.2.6-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_cedarview-6.2_4.2.6-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_4.2.6-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_qoriq-6.2_4.2.6-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_rtd1296-6.2_4.2.6-1.spk

DONCIEUX-CAMILLE committed:
Update README.md

DONCIEUX-CAMILLE committed:
Zabbix 4.4.0 for Arch Qoriq 6.2

https://www.zabbix.com/whats_new_4_4

DONCIEUX-CAMILLE committed:
Zabbix 4.4.0 for Arch Cedarview 6.2

https://www.zabbix.com/whats_new_4_4

DONCIEUX-CAMILLE committed:
Zabbix 4.4.0 for Arch Apollolake 6.2

https://www.zabbix.com/whats_new_4_4

DONCIEUX-CAMILLE committed:
Zabbix 4.4.0 for Arch RTD1296 6.2

https://www.zabbix.com/whats_new_4_4

DONCIEUX-CAMILLE committed:
Zabbix 4.4.0 for Arch Evansport 5.2

https://www.zabbix.com/whats_new_4_4

DONCIEUX-CAMILLE committed:
Update README.md

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch Denverton 6.1

DONCIEUX-CAMILLE committed:
Delete zabbix_evansport-5.2_4.4.0-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_apollolake-6.2_4.4.0-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_armada38x-6.2_4.4.0-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_cedarview-6.2_4.4.0-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_qoriq-6.2_4.4.0-1.spk

DONCIEUX-CAMILLE committed:
Delete zabbix_rtd1296-6.2_4.4.0-1.spk

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch armada38x 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch Apollolake 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch Cedarview 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch qoriq 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch rtd1296 6.2

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch Evansport 5.2

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch Broadwell 6.2

DONCIEUX-CAMILLE commented:
> Hi
> is it possible to have the Proxy agent (RTD and ARM) ?
> Thanks

Sorry I haven't develop inside the package the proxy part as Zabbix server and Proxy on the same synology server will lead to big cpu consumptions. 
Do you have a Zabbix server elsewhere ?
Camille.

DONCIEUX-CAMILLE committed:
Zabbix 4.4.4 with arch Avoton 6.2

DONCIEUX-CAMILLE commented:
> Hello, i would like to use Zabbix on DS2415+ with Avoton architecture. Is there a branch available?
> Many thanks in advance!
Please have a look on
https://github.com/DONCIEUX-CAMILLE/zabbix-spk
I've committed your architecture
Zabbix 4.4.4 with arch Avoton 6.2
I will now provide for you next Zabbix update.
Camille

DONCIEUX-CAMILLE commented:
Hello
@Peda86
Done.

@puzzle-it-nu 
Could you close the pull request please
Thanks
Camille

DONCIEUX-CAMILLE committed:
Update README.md

Wrong branch in your repository
@mcdull
@nckdnhm
@jborg
@thetuxinator
I have noticed that you forked an old branch of my repository (by mistake I guess).
Could you please delete your branch as the new one is now the 4.4.4 version and create pull request instead if you need upadte in the package.
Each time that Zabbix SA is providing an update, the actual branch will become obsolete too.
As the 4.4.5 was publihed on January 29th 2020, the according new branch will be provided as soon as possible with proxy support as requested in https://github.com/puzzle-it-nu/zabbix-spk/pull/10 and will become the default branch of my repository.

Thanks.
Camille.

DONCIEUX-CAMILLE committed:
Update README.md
